app/browser/ads/adBlockUtil.js

```diff
@@ -31,16 +31,27 @@ const mapFilterType = {
 const shouldDoAdBlockCheck = (resourceType, firstPartyUrl, url, shouldCheckMainFrame) =>
```

Comment:
cc @darkdh 

@diracdeltas I am not familiar with this code, so any feedback and help is more then welcome